runner/metrics/metrics.go

```diff
@@ -0,0 +1,70 @@
+package metrics
+
+import (
+	"log/slog"
+	"time"
+
+	"github.com/cloudbase/garm/auth"
+	"github.com/cloudbase/garm/runner"
+)
+
+func CollectObjectMetric(r *runner.Runner, timer *time.Ticker) {
```

Comment:
This function should accept a context. Then we can pass in the global context that gets generated in `main()`. The reason is that we need to end the goroutine bellow when `ctx.Done()` returns.

Reply:
done with https://github.com/cloudbase/garm/pull/216/commits/97f172eb513bda47d6658b81fd9bf262d687aa98